pyt/__main__.py

```diff
@@ -298,15 +301,38 @@ def main(command_line_args=sys.argv[1:]):
 
     analyse(cfg_list, analysis_type=analysis)
 
+    nosec_lines = set()
```

Comment:
Can you move this code
```python
    if args.ignore_nosec:
        nosec_lines = set()
    else:
        file = open(path, "r")
        lines = file.readlines()
        nosec_lines = set(
                    lineno for
                    (lineno, line) in enumerate(lines, start=1)
                    if '#nosec' in line or '# nosec' in line) 
```
to near the top, so we take into account `nosec_lines` at both call-sites to find_vulnerabilities.
https://github.com/omergunal/pyt/blob/fb88051e1d988d5890127ef3aa6867adf0db07de/pyt/__main__.py#L240
is a good spot, the same way `UImode` is set once and then passed to both call-sites https://github.com/python-security/pyt/blob/master/pyt/__main__.py#L232-L236